skills/lark-doc/scripts/doc_word_stat.py

```
@@ -0,0 +1,1125 @@
+        explicit = elem.attrib.get("type")
+    if explicit:
+        return TYPE_ALIASES.get(explicit, explicit)
+    return TYPE_ALIASES.get(tag, tag)
+
+
+def parse_xml(source: str) -> list[Block]:
+    source = source.strip()
+    if not source:
+        return []
+    try:
+        root = ET.fromstring(source)
```

Ruff (0.15.20)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)